### ISSUE TITLE
Encrypted but not really _encrypted_ PDF isn't parsed and detected as "secured pdf"

### ISSUE THREAD

**Issue by rameezrami** [smalot/pdfparser#320]
This is a very good package with excellent and easy to use API but it seems not production-ready yet. I'm having errors with different PDFs. 

also, I'm getting an error saying pdf in encrypted. I double-checked if the PDF is protected, it wasn't.

maybe a version issue? like in another issue posted in this repository.

I have created a screencast for better understanding and these pdf is not getting parsed in your demo page too.

Please find the files in the attachment.

[pdf and screencasts.zip](https://github.com/smalot/pdfparser/files/4936225/pdf.and.screencasts.zip)

**Comment by cesardmoro** [comment]
any advice on this  ?

**Comment by k00ni** [comment]
> any advice on this ?

Sorry, I can't help you on this one.

**Comment by j0k3r** [comment]
I've tried locally and on my Mac the PDF information are saying the PDF is password encrypted.

<img width="255" src="https://user-images.githubusercontent.com/62333/90859012-a2d24200-e387-11ea-9907-c30164059563.png">

Also tried using an online tool and it says the same: https://www.metadata2go.com/result/cffe1bdf-d6be-45d1-b810-f4db8ae846af

![image](https://user-images.githubusercontent.com/62333/90859109-d745fe00-e387-11ea-86fe-7c44599067c4.png)

**Comment by rameezrami** [comment]
@j0k3r but you were able to open the pdf without any issue right? no passwords asked. I have a couple of other pdfs(can't share here, confidential information) too which doesn't work the same way.

**Comment by j0k3r** [comment]
Of course, but the label says the PDF is encrypted. So we don't try to read it at all.
See https://github.com/smalot/pdfparser/blob/master/src/Smalot/PdfParser/Parser.php#L97-L99

Maybe we should improve that and try to read the PDF?

**Comment by Sagar1219** [comment]
Hey I have encountered same issue, is this fixed? Also, meanwhile do we have any alternate solution for this ?

**Comment by j0k3r** [comment]
No alternate solution, but contributions are welcome.

**Comment by unixnut** [comment]
Fix in _unixnut/pdfparser_ fork, 'decryption' branch.  I was going to create a draft pull request but I already have an unrelated PR open for one of the commits on that branch so I can't make another until that's merged.  @k00ni can you please reopen this issue?

**Comment by matkozikowski** [comment]
Hi,
Do you know if there is pull request of continue work with `src/Smalot/PdfParser/Config::setIgnoreEncryption` ? I would like to use this but if is depreacted do we have some alternative method to use ?

**Comment by unixnut** [comment]
@matkozikowski @j0k3r @k00ni I am still interested in making a PR for my decryption branch.  However, I have been struggling emotionally and need to get into counselling, which should happen soon.

**Comment by k00ni** [comment]
@unixnut I wish you all the best.

**Comment by unixnut** [comment]
@k00ni I sent you an e-mail about this project.  No rush to reply, but if you didn't see it can you please check your spam folder?  Thanks.
